### PR TITLE
Working

### PR DESCRIPTION
**Pull request by BSGreaves**
Done!

### CODE REVIEW

**Comment by BSGreaves** [comment]
@drdreed
Done with this one too. Sorry, forgot to tag you in these. I'll do so now. (I did the assignment before she told us to tag you guys).

File: main.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+//Find the lowest grade
+var lowestGrade = scores;
+for (i = 0; i < lowestGrade.length; i++) {
+	lowestGrade.sort(function(a, b){return a-b});
+}
+console.log("The lowest grade is ", lowestGrade[0]);
+
+//Find the highest grade
+var highestGrade = scores;
+for (i = 0; i < highestGrade.length; i++) {
```
Comment (DevDreed) [inline main.js]: for loop is not needed.  You are not using the variable `i` inside the loop.

File: main.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+console.log("Number of F's: ", fGrade.length);
+console.log("Number of D's: ", dGrade.length);
+console.log("Number of C's: ", cGrade.length);
+console.log("Number of B's: ", bGrade.length);
+console.log("Number of A's: ", aGrade.length);
+console.log("Number of failing grades: ", fail.length);
+
+//Find the lowest grade
+var lowestGrade = scores;
+for (i = 0; i < lowestGrade.length; i++) {
```
Comment (DevDreed) [inline main.js]: for loop is not needed.  You are not using the variable `i` inside the loop.

**Review by DevDreed — CHANGES_REQUESTED** [review]
Looking good.  The array.sort() method does not need to be inside the `for` loop.

**Comment by BSGreaves** [comment]
Removed the loops, plus a test array I didn't need. Thanks! Don't know what I was thinking. Should be ready to go now.

**Review by DevDreed — APPROVED** [review]
👍

**Pull request merged**